### PR TITLE
Update external control plane installation docs

### PR DESCRIPTION
**Pull request by frankbu**
1. Added explanatory text, and verification steps.
2. Restructured to separate instructions by role (mesh operator / mesh admin).
3. Uses template injection to start gateways (a mesh admin role now).
4. Include multicluster external control plane instructions.

### CODE REVIEW

File: content/en/docs/setup/additional-setup/external-controlplane/index.md
Comment (frankbu) [inline content/en/docs/setup/additional-setup/external-controlplane/index.md:448]: This command will currently fail. Latest master build fails to inject the gateway template because the deployment has `istio.io/rev: default` (instead of no `istio.io/rev` label), which the sidecar-injector webhook doesn't currently allow.

@Monkeyanator I understand (from @howardjohn) that you have an upcoming fix for this. Is that correct?

Comment (Monkeyanator) [reply content/en/docs/setup/additional-setup/external-controlplane/index.md:448]: Yes, that's correct. I was going to hold off until the default revision work merged, but since that might be delayed a bit, I'll get an implementation up ASAP

Comment (frankbu) [reply content/en/docs/setup/additional-setup/external-controlplane/index.md:448]: Great, thanks!

File: content/en/docs/setup/additional-setup/external-controlplane/index.md
Comment (frankbu) [inline content/en/docs/setup/additional-setup/external-controlplane/index.md:166]: Would it be possible for `istioctl install` to do this automatically, just like it does for local istio-system ns, so we can eliminate this step?

File: content/en/docs/setup/additional-setup/external-controlplane/index.md
Comment (howardjohn) [reply content/en/docs/setup/additional-setup/external-controlplane/index.md:448]: it should work with https://github.com/istio/istio/pull/31344 anyways?

Comment (frankbu) [reply content/en/docs/setup/additional-setup/external-controlplane/index.md:448]: Oh, yes @howardjohn, looks like it is already working with #31344. Thanks.

**Review by ericvn — APPROVED** [review]

**Pull request merged**
